### DISCUSSION TITLE
6.31 CN RSV

### DISCUSSION THREAD

**Discussion by Souma-Sumire** [GpointChen/FFXIVChnTextPatch-GP#354] [Show and tell]
我不确定中国服的ID格式如何修改使其适用于国际服的ID格式，例如InstanceContentTextData的35827，就不提交PR了。
[6.31RSV.txt](https://github.com/GpointChen/FFXIVChnTextPatch-GP/files/11801020/6.31RSV.txt)

**Comment by GpointChen** [comment]
感謝提交，已更新漢化。

**Comment by Souma-Sumire** [comment]
學習了巴哈的筆記，`<hex:022C0DFF07><hex:022903EB0203><hex:FF02> <hex:0203>`應該是國際服的ID（名字部分）？
